.github/workflows/CD.yaml

```diff
@@ -29,11 +29,11 @@ jobs:
           job: ${{ toJSON(job) }}
           steps: ${{ toJSON(steps) }}
           runner: ${{ toJSON(runner) }}
-          secrets: ${{ toJSON(secrets) }}
           strategy: ${{ toJSON(strategy) }}
           matrix: ${{ toJSON(matrix) }}
           needs: ${{ toJSON(needs) }}
           inputs: ${{ toJSON(inputs) }}
+          vars: ${{ toJSON(vars) }}
```

Comment:
What is the difference between github secrets and vars in terms of being visible in the logs?